src/Events/Calendar_Embeds/Admin/List_Table.php

```diff
@@ -0,0 +1,132 @@
+<?php
+/**
+ * Calendar Embed List Table
+ *
+ * @since TBD
+ *
+ * @package TEC\Events\Calendar_Embeds\Admin
+ */
+
+namespace TEC\Events\Calendar_Embeds\Admin;
+
+use TEC\Events\Calendar_Embeds\Calendar_Embeds;
+use Tribe__Template;
+use Tribe__Events__Main;
+
+/**
+ * Class List_Table
+ *
+ * @since TBD
+ *
+ * @package TEC\Events\Calendar_Embeds\Admin
+ */
+class List_Table {
+	/**
+	 * The template.
+	 *
+	 * @since TBD
+	 *
+	 * @var Tribe__Template
+	 */
+	protected $template;
+
+	/**
+	 * Customize columns for the table.
+	 *
+	 * @since TBD
+	 *
+	 * @param array $columns The columns.
+	 *
+	 * @return array
+	 */
+	public function manage_columns( $columns ): array {
+		$new_columns = [
+			'cb'               => $columns['cb'] ?? '<input type="checkbox" />',
+			'title'            => __( 'Calendar Embeds', 'the-events-calendar' ),
+			'event_categories' => __( 'Categories', 'the-events-calendar' ),
+			'event_tags'       => __( 'Tags', 'the-events-calendar' ),
+			'snippet'          => __( 'Embed Snippet', 'the-events-calendar' ),
+		];
+
+		/**
+		 * Filters the columns for the calendar embeds list table.
+		 *
+		 * @since TBD
+		 *
+		 * @param array $new_columns The columns.
+		 *
+		 * @return array The filtered columns.
+		 */
+		return (array) apply_filters( 'tec_events_calendar_embeds_list_table_columns', $new_columns );
+	}
+
+	/**
+	 * Customize the content of the columns.
+	 *
+	 * @since TBD
+	 *
+	 * @param string $column_name The name of the column.
+	 * @param int    $post_id     The post ID.
+	 *
+	 * @return void
+	 */
+	public function manage_column_content( $column_name, $post_id ): void {
+		switch ( $column_name ) {
+			case 'event_categories':
+				// Get events categores from post meta.
```

Comment:
```suggestion
				// Get events categories from post meta.
```
spellcheck on aisle 4 😉